unit_tests/tests/sensor/test_flex_sensor.cpp

```diff
@@ -0,0 +1,148 @@
+#include "flex_sensor.h"
+#include "engine_test_helper.h" // Used for timesourcing
```

Comment:
you can replace all these includes with `#include "pch.h"`, except for flex_sensor.h

Reply:
Noted